skyportal/plot.py

```diff
@@ -687,7 +687,212 @@ def photometry_plot(obj_id, user, width=600, height=300):
 
     p2 = Panel(child=layout, title='Mag')
 
-    tabs = Tabs(tabs=[p2, p1], width=width, height=height, sizing_mode='fixed')
+    # now make period plot
+
+    if obj.varstar:
+
+        # get period from annotations (any for now)
+        annotation_list = obj.get_annotations_readable_by(user)[0]
+        if 'period' in annotation_list.data:
+            period = annotation_list.data['period']
+        else:
+            period = 0.1
+
+        # bokeh figure for period plotting
+        period_plot = figure(
+            aspect_ratio=1.7,
+            sizing_mode='scale_both',
+            active_drag='box_zoom',
+            tools='box_zoom,wheel_zoom,pan,reset,save',
+            y_range=(ymax, ymin),
+            x_range=(-0.1, 2.1),  # cover two phases: a and b
+            toolbar_location='above',
+            toolbar_sticky=False,
+            x_axis_location='below',
+        )
+
+        # axis labels
+        period_plot.xaxis.axis_label = 'phase'
```

Comment:
```suggestion
        period_plot.xaxis.axis_label = r'phase ($\phi$) [radian]'
```

Reply:
Not familiar with this labeling.  I couldn't find any examples of this in the literature.  I don't believe radians is the correct unit since a full cycle should be 2 pi radians, no?

Reply:
the phase is in units of radians. A full cycle is indeed from 0 to 2pi. Phase going from 0-1 is sometimes shown but I do not think that it is as intuitive as radian units.

Reply:
Can you find me an example of this notation in the literature?  I cannot find any such example.  Uniformly, researchers in periodic variables use simple phase.  I worry that adopting a non-standard notation would confuse people, regardless of its correctness.